tests/utils.py

```diff
@@ -19,7 +23,45 @@ def unorder(value):
     return unorder(xmltodict.parse(xml))
 
 
+NAMESPACE_ATTRIBUTE = re.compile(' xmlns="[^"]+"', flags=re.UNICODE)
+
+
+class ParsedXml(object):
+
+    def __init__(self, xml_string):
+        if isinstance(xml_string, six.binary_type):
```

Comment:
Maybe document somewhere that any binary data in the xml string will be decoded as 'utf-8'.